python/packages/nisar/workflows/rubbersheet.py

```diff
@@ -22,9 +24,197 @@
+
+                data = np.column_stack([
+                    np.arange(corr_peak.size),
+                    np.repeat(az_polyfit_indices, len(rg_polyfit_indices)),
+                    np.tile(rg_polyfit_indices, len(az_polyfit_indices)),
+                    az_offsets.ravel(), rg_offsets.ravel(),
+                    corr_peak.ravel()
+                ])
+
+                # polyfitting the range and azimuth offsets
+                results = offsets_polyfit.polyfit_offsets(
+                    data,degree=rubbersheet_params['polyfitting']['degree'],
+                    crit_value=rubbersheet_params['polyfitting']['critical_value'],
+                    max_iterations=len(data),
+                    minL=minL, maxL=maxL,
+                    minP=minP, maxP=maxP,
+                    prf=prf, rbw=rbw,
+                    abw=abw, rsr=rsr)
+
+                polyfitting_degree = results["degree"]
+                polyfitting_coefL = results["coefL"]
+                polyfitting_coefP = results["coefP"]
+
+                print(f"Polyfitting Degree: {polyfitting_degree}")
+                print(f"Polyfitting CoefL: {polyfitting_coefL}")
+                print(f"Polyfitting CoefP: {polyfitting_coefP}")
+
+                info_channel.log(f"Polyfitting Degree: {polyfitting_degree}")
+                info_channel.log(f"Polyfitting CoefL: {polyfitting_coefL}")
+                info_channel.log(f"Polyfitting CoefP: {polyfitting_coefP}")
+
+                lines, pixels = np.meshgrid(off_az_indices, off_rg_indices, indexing="ij")
+                culled_az_offsets, culled_rg_offsets =\
+                    offsets_polyfit.predict_offsets_batch(lines, pixels,
+                                                          results["coefL"],
+                                                          results["coefP"],
+                                                          results["degree"],
+                                                          minL, maxL, minP, maxP)
+
+                _write_to_disk(str(f'{out_dir}/culled_az_offsets'),culled_az_offsets)
+                _write_to_disk(str(f'{out_dir}/culled_rg_offsets'),culled_rg_offsets)
+
+                # Get ground velocity and correlation peak
+                ground_track_velocity = _open_raster(ground_track_velocity_file)
+
+                # Get datasets from HDF5 file and update datasets
+                offset_az_prod = dst_h5[f'{pol_group_path}/alongTrackOffset']
+                offset_rg_prod = dst_h5[f'{pol_group_path}/slantRangeOffset']
+                offset_peak_prod = dst_h5[f'{pol_group_path}/correlationSurfacePeak']
+
+                # Assign cross-correlation peak
+                offset_peak_prod[...] = _open_raster(corr_peak_path)
+                # Convert the along track and slant range pixel offsets to meters
+                offset_az_prod[...] = \
+                    culled_az_offsets * ground_track_velocity \
+                    * ref_zero_doppler_time_spacing
```

Comment:
```suggestion
                offset_az_prod[...] = (
                    culled_az_offsets
                    * ground_track_velocity
                    * ref_zero_doppler_time_spacing
                )
```
Multi-line formulae can be done like this, saves you the effort of putting a bunch of back slashes and is a little easier to follow at a glance